domain/src/main/java/no/difi/meldingsutveksling/core/EDUCoreFactory.java

```diff
@@ -127,6 +144,127 @@ public EDUCore create(Message message, String senderOrgNr) {
         return eduCore;
     }
 
+    public EDUCore create(ConversationResource cr, Arkivmelding am, byte[] asic) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "create" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)